### ISSUE TITLE
Incompatibility Issue between Bun and Next-Export-Optimize-Images with Sharp Module

### ISSUE THREAD

**Issue by holykzm** [dc7290/next-export-optimize-images#733]
### Describe the bug
When building a Next.js project using `bun`, an incompatibility issue arises between `next-export-optimize-images` and the `sharp` module. Specifically, an error occurs during the installation of the `sharp` module, leading to the interruption of the build process.

### To Reproduce

1. Run `bun run build` in the terminal.
1. Observe that the command `next build && next-export-optimize-images` is executed.
1. Notice the error during the build process.

### Expected behavior
The `sharp` module should install correctly, and the build process should complete successfully.

### Error Log

```shell
Error: 
〜/node_modules/next-export-optimize-images/node_modules/sharp/lib/sharp.js:37
  throw new Error(help.join('\n'));
  
Something went wrong installing the "sharp" module

Cannot find module '../build/Release/sharp-darwin-arm64v8.node'
Require stack:
- ~/pakistan-pjh.com/node_modules/next-export-optimize-images/node_modules/sharp/lib/sharp.js
- ~/pakistan-pjh.com/node_modules/next-export-optimize-images/node_modules/sharp/lib/constructor.js
...

Possible solutions:
- Install with verbose logging and look for errors: "npm install --ignore-scripts=false --foreground-scripts --verbose sharp"
- Install for the current darwin-arm64v8 runtime: "npm install --platform=darwin --arch=arm64v8 sharp"
- Consult the installation documentation: https://sharp.pixelplumbing.com/install
...

Node.js v18.16.1
error: script "build" exited with code 1 (SIGHUP)

```

### Desktop (please complete the following information):

OS: macOS Sonoma 14.1.1 (m1)
Node.js Version: 18.16.1
Bun Version: 1.0.15

### Additional context
This issue seems specific to the use of `bun` for building, as the build process completes successfully when using traditional Node.js tools like `npm` or `yarn`. 🧐

**Comment by arctixdev** [comment]
I think this was fixed in sharp 0.33. If you add 
```
  "overrides": {
    "sharp": "0.33.1"
  }
```
To your package.json it should fix it. But it would be verry nice if next-export-optimize-images would update its sharp dependency to the latest to fix this.

**Comment by holykzm** [comment]
@arctixdev 

Thank you for your suggestion to add "sharp": "0.33.1" to the overrides. I tried this, but unfortunately, it didn’t work out in my environment. I was able to resolve the issue by downgrading to "sharp": "0.32.6". I hope this information might help others facing similar issues.
